container/attributes.go

```diff
@@ -12,7 +12,66 @@ const (
 
 	// SysAttributeZone is a string of zone for container name.
 	SysAttributeZone = SysAttributePrefix + "ZONE"
+
+	// SysAttributeHomomorphicHashing is a container's homomorphic hashing state.
+	SysAttributeHomomorphicHashing = SysAttributePrefix + "HOMOMORPHIC-HASHING"
```

Comment:
It is a bit strange to consider `HOMORPHIC-HASHING` as `false` when it is missing. But we need this for backwards compatibility. What about `DISABLE-HOMOMORPHIC-HASHING`?